PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,15 +1,15 @@
-| Q                     | A
-| --------------------- | ---
-| ⏱ Review tijd        | <!-- 5 min -->
-| ⌨️ Type wijziging      | <!-- 🛠 Bug fix / ➕ Nieuwe feature / 🗒 Docs / 🤷🏻‍♂️ Overige -->
+| Q | A
+| -| -
+| ⏱ Review tijd | 5 min
+| ⌨️ Type wijziging | Bug fix / Nieuwe feature / Docs / Overige
 
```

Comment:
```suggestion
| Q              | A
| ---------------| -
| Review tijd    | 5 min
| Type wijziging | Bug fix / Nieuwe feature / Docs / Overige
```

Reply:
Dit resulteert dus in spacing issues, want het stond al zo voorheen.

Reply:
In deze repo is monospace, in het PR zelf is het geen monospace.